docs/src/main/sphinx/release/release-445.md

```diff
@@ -0,0 +1,33 @@
+# Release 445 (10 Apr 2024)
+
+## General
+
+* Improve performance of queries selecting only catalog, schema, and relation
+  name from the `system.metadata.materialized_views` table. ({issue}`21448`)
+* Fix potential query hang when there is an error processing data. ({issue}`21397`)
+
+## Delta Lake connector
+
+* Add support for [time travel](delta-time-travel) queries. ({issue}`21052`)
+
+## Hive connector
+
+* {{breaking}} Remove the deprecated `PARTITION_COLUMN` and `PARTITION_VALUE`
+  arguments from the `flush_metadata_cache` procedure in favor of
+  `PARTITION_COLUMNS` and `PARTITION_VALUES`. ({issue}`21410`)
+
+## MySQL connector
+
+* Improve performance of queries with `timestamp(n)` values. ({issue}`21244`)
+
+## PostgreSQL connector
+
+* Improve performance of queries with `timestamp(n)` values. ({issue}`21244`)
+
+## Redis connector
+
+* Upgrade minimum required Redis version to 5.0.14 or later. ({issue}`21455`)
+
+## Snowflake connector
+
+* Add support for pushdown of statistical aggregation. ({issue}`21384`)
```

Comment:
Thats a thing?

Reply:
please name the functions explicitly since not all statistical aggregation functions are supported.

Reply:
Yeah .. also .. did the docs get updated with that PR?

Reply:
Yes updated. the functions are `variance`, `var_pop`,`var_samp`,`covar_pop`,`covar_samp`, `corr`,`regr_intercept`,`regr_slope`